languages/common-lisp/exercises/concept/lillys-lasagna/lillys-lasagna-test.lisp

```diff
@@ -0,0 +1,51 @@
+;; Ensures that lillys-lasagna.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (load "lillys-lasagna")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from lillys-lasagna and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage :lillys-lasagna-test
+  (:use :cl :fiveam :lillys-lasagna)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :lillys-lasagna-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite lillys-lasagna-suite)
+(in-suite lillys-lasagna-suite)
+
+(test expected-time "Compute the expected time in the oven"
+  (is (= 337 (expected-time-in-oven)))
+  (is (equal "Number of minutes Lasagna should be in oven."
```

Comment:
If we are testing for specific doc-strings (as opposed to them just being there), we might want to put that in the instructions? I think someone would need to see the tests to pass them at the moment. While that's something they should be able to do, it might not be the ideal workflow.